src/store/api/index.js

```diff
@@ -1,6 +1,15 @@
 import { fallbackApolloClient } from '@/vue-apollo'
 import LogRocket from 'logrocket'
 
+const SERVER_KEY = `${process.env.VUE_APP_RELEASE_TIMESTAMP}_server_url`
```

Comment:
Not a big deal but will this end up filling local storage with lots of unused keys?

Reply:
I think it should be ok - this can always be cleared by a user (as part of a broader browser cache clear) but if not we use localStorage sparingly enough elsewhere that this shouldn't have an impact. Good callout though, let's keep an eye on it and find a more permanent solution if needed.